keyboards/yushakobo/quick7/rules.mk

```
@@ -0,0 +1,44 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
```

If this board runs on a Pro Micro by default, this should be

```suggestion
BOOTLOADER = caterina
```

If you're using an on-board microcontroller then this should remain unchanged.